# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/ContentParent.cpp
# Commit: bde8ca8fd963
# Full Hash: bde8ca8fd9636bc6650ee6768dfb8e60fc1da596
# Author: Kagami Sascha Rosylight <saschanaz@outlook.com>
# Date: 2024-11-12 09:32:50
# Regressor Bug: 1928702
# File Overlap Count: 1
# Description:
#   Bug 1928702 - Part 6: Move NotificationObserver functions to NotificationUtils r=asuth
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D227691
# ==============================================================================

diff -r fca3921d09a5 -r bde8ca8fd963 dom/ipc/ContentParent.cpp
--- a/dom/ipc/ContentParent.cpp	Mon Nov 11 20:34:13 2024 +0000
+++ b/dom/ipc/ContentParent.cpp	Mon Nov 11 20:34:13 2024 +0000
@@ -113,7 +113,7 @@
 #include "mozilla/dom/MediaController.h"
 #include "mozilla/dom/MemoryReportRequest.h"
 #include "mozilla/dom/MediaStatusManager.h"
-#include "mozilla/dom/Notification.h"
+#include "mozilla/dom/notification/NotificationUtils.h"
 #include "mozilla/dom/PContentPermissionRequestParent.h"
 #include "mozilla/dom/PCycleCollectWithLogsParent.h"
 #include "mozilla/dom/ParentProcessMessageManager.h"
@@ -4725,7 +4725,7 @@
   if (!ValidatePrincipal(aPrincipal)) {
     LogAndAssertFailedPrincipalValidationInfo(aPrincipal, __func__);
   }
-  Unused << Notification::RemovePermission(aPrincipal);
+  Unused << notification::RemovePermission(aPrincipal);
   return IPC_OK();
 }
 
@@ -4738,7 +4738,7 @@
   if (!ValidatePrincipal(aPrincipal)) {
     LogAndAssertFailedPrincipalValidationInfo(aPrincipal, __func__);
   }
-  Unused << Notification::OpenSettings(aPrincipal);
+  Unused << notification::OpenSettings(aPrincipal);
   return IPC_OK();
 }
 